File: Domain/Tretmani.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;

namespace Domain
{
    public class Tretmani
    {
        public int Id { get; set; }
        public string Emri { get; set; }
        public string Pershkrimi { get; set; }
        public float Cmimi { get; set; }
        public string DokoriId { get; set; }
        public Doktori Doktori { get; set; }
        public Pagesa Pagesa { get; set; }
        public string PacientId { get; set; }
        public Pacienti Pacienti { get; set;}
        public int KontrollaId { get; set; }
        public Kontrolla Kontrolla { get; set; }
        public ICollection<Udhezimi> Udhezimet { get; set; } = new List<Udhezimi>();
    }
}

File: client-app/src/app/models/tretmani.ts
import { Doktori } from "./doktori";
import { Kontrolla } from "./kontrolla";
import { Pacienti } from "./pacienti";
import { Pagesa } from "./pagesa";
import { Udhezimi } from "./udhezimi";
import { XRay } from "./xray";

export interface Tretmani {
    id?: string;
    emri: string;
    pershkrimi: string;
    cmimi: string;
    doktoriId?: string;
    doktori?: Doktori;
    pagesaId?: string;
    pagesa?: Pagesa;
    pacientId: string;
    pacienti?: Pacienti;
    kontrollaId?: string;
    kontrollat?: Kontrolla[];
    udhezimet?: Udhezimi[];
    xrays?: XRay[];
}